coderadar-server/coderadar-core/src/test/java/org/wickedsource/coderadar/job/scan/commit/CommitMetadataScannerIntegrationTest.java

```diff
@@ -79,22 +74,23 @@ public void cleanup() {
   }
 
   @Test
-  @Category(IntegrationTest.class)
+  @Tag("IntegrationTest.class")
```

Comment:
same as with "ControllerTest.class"

Reply:
Done

Reply:
@maximAtanasov I see no change

Reply:
Sorry, I missed that tag. I've changed it to simply `IntegrationTest` now.